include/rdma/rdma_cma.h

```diff
@@ -197,6 +197,29 @@ struct rdma_addrinfo {
 	struct rdma_addrinfo	*ai_next;
 };
 
+/* Multicast join compatibility mask attributes */
+enum rdma_cm_join_mc_attr_mask {
+	RDMA_CM_JOIN_MC_ATTR_ADDRESS	= 1 << 0,
+	RDMA_CM_JOIN_MC_ATTR_JOIN_FLAGS	= 1 << 1,
+	RDMA_CM_JOIN_MC_ATTR_RESERVED	= 1 << 2,
+};
+
+/* Multicast join flags */
+enum rdma_cm_mc_join_flags {
+        RDMA_MC_JOIN_FLAG_FULLMEMBER,
+        RDMA_MC_JOIN_FLAG_SENDONLY_FULLMEMBER,
+        RDMA_MC_JOIN_FLAG_RESERVED,
+};
```

Comment:
Will remove the name from these enums. The join flags
are corresponding to the join flags in the kernel under
rdma_user_cm.h "Multicast join flags", this will require a
kernel change. Regarding the bit value unlike the comp mask
which should be orthogonal it is not clear why it should be changed.